pkg/driver/controller.go

```diff
@@ -115,19 +115,6 @@ func (d *controllerService) CreateVolume(ctx context.Context, req *csi.CreateVol
 	}
 	defer d.inFlight.Delete(volName)
 
-	disk, err := d.cloud.GetDiskByName(ctx, volName, volSizeBytes)
```

Comment:
With the volumeName in the ClientToken, we no longer need describeVolume to check if the volume has already been created or not. But we will also lose some visibility to customer, if the two createVolume requests have same clientToken(volumeName) but different parameters (size or snapshotId), ebs api will return general idempotency violation exception. (instead of saying diff size or diff snapshotId)

Open to any idea, if you want me to add it back

Reply:
i'm good with removing this code